api.js

```diff
@@ -145,7 +145,8 @@ Api.prototype._run = function (files, options) {
 	var runStatus = new RunStatus({
 		runOnlyExclusive: options.runOnlyExclusive,
 		prefixTitles: this.options.explicitTitles || files.length > 1,
-		base: path.relative('.', commonPathPrefix(files)) + path.sep
+		base: path.relative('.', commonPathPrefix(files)) + path.sep,
+		failFast: options.failFast
```

Comment:
Use `this.options.failFast` instead and you don't have to pass it on https://github.com/avajs/ava/pull/1160/files#diff-f8849ae9f0614ba9464fdd14c2d98dc0R163 as it's already passed in the API constructor.